src/snowflake/cli/plugins/sql/manager.py

```diff
@@ -26,7 +27,7 @@ def execute(
         if std_in:
             query = sys.stdin.read()
         elif file:
-            query = file.read_text()
+            query = SecurePath(file).read_text(file_size_limit_mb=128)
```

Comment:
IMO SQL files shouldn't have a size limit. They can contain DB dumps which can be large.

Reply:
Done